k8s/crds/kops.k8s.io_clusters.yaml

```diff
@@ -5156,6 +5156,10 @@ spec:
               ntp:
                 description: NTPConfig is the configuration for NTP.
                 properties:
+                  chrony:
```

Comment:
Maybe a string `provider: chrony` would be more extensible?  (And it could serve as a "discriminated union" field https://github.com/kubernetes/community/issues/229 )

Reply:
I was hoping to remove it later. It is only useful for Ubuntu 20.04 that will be removed in the next few years. I doubt anything exciting will happen with NTP by then.